src/main/java/com/revature/services/EvalLogicImpl.java

```diff
@@ -16,40 +16,76 @@
 
 	@Autowired
 	private EvalRepository dao;
+	
+
+//CREATE----------------------------
 
+//RETRIEVE--------------------------
 	@Override
 	public Eval getEvalById(int id) {
 		Eval eval = dao.findOne(id);
-		System.out.println(eval);
+		try {
+			System.out.println(eval);
+		} catch (Exception e) {e.printStackTrace();}
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1148)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1166)